packages/api/scripts/verify-approval-canonical-vectors.py

```
@@ -128,17 +128,21 @@ def pointer_tokens(pointer: str) -> list[str]:
-    result = copy.deepcopy(source)
-    tokens = pointer_tokens(pointer)
-    target = result
-    for token in tokens[:-1]:
-        target = target[int(token)] if isinstance(target, list) else target[token]
-    final = tokens[-1]
-    if isinstance(target, list):
-        target[int(final)] = replacement
-    else:
-        target[final] = replacement
-    return result
+    # 越界 JSON Pointer 归一成结构化 ValueError，避免裸 IndexError traceback。
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)